ssl/ssl_local.h

```diff
@@ -2819,6 +2819,17 @@ int tls_provider_set_tls_params(SSL *s, EVP_CIPHER_CTX *ctx,
                                 const EVP_CIPHER *ciph,
                                 const EVP_MD *md);
 
+void ssl_engine_finish(ENGINE *e);
+const EVP_CIPHER *tls_get_cipher_from_engine(int nid);
+const EVP_MD *tls_get_digest_from_engine(int nid);
+int tls_engine_load_ssl_client_cert(SSL *s, X509 **px509, EVP_PKEY **ppkey);
+int ssl_old_hmac_new(SSL_HMAC *ret);
```

Comment:
Maybe change to ssl_hmac_old_new, ssl_hmac_old_free.... etc. as the context parameter is SSL_HMAC type.

Reply:
Done